### DISCUSSION TITLE
Output folder does not generate after build

### DISCUSSION THREAD

**Discussion by m0ksem** [nuxt/nuxt#17941] [Questions] [answered]
# Desc
After running `yarn build` it does not generate any output folder. Changing output dir does not help too.
```
 nitro: {
    output: {
      dir: 'builded'
    }
  },
```
Command can not be finished and can be only interupted...

As I can see people have similar issue on Mac, with npm or yarn, but I can not find way to fix that.
```vite: false,``` does not help too.

`yarn dev` works fine.

# Env
WSL Ubuntu 18.04.5 LTS

Fresh project, created using `npx nuxi init nuxt3-app`
```
- Operating System: `Linux`
- Node Version:     `v14.19.0`
- Nuxt Version:     `3.0.0-27409835.c53c736`
- Package Manager:  `yarn@1.22.17`
- Bundler:          `Vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
```
# Related issues
https://github.com/nuxt/framework/issues/2259 - Using latest esbuild version does not help.
https://github.com/nuxt/framework/issues/2256 - I don't have any modules to transpile

yarn build output

```
✔ Server Compiled successfully in 2.14s                                                                                                                                              00:18:45
✔ Client Compiled successfully in 3.10s                                                                                                                                              00:18:46
✔ Client Compiled successfully in 344.08ms                                                                                                                                           00:18:46
✔ Server Compiled successfully in 395.29ms                                                                                                                                           00:18:46
✔ Client Compiled successfully in 287.84ms                                                                                                                                           00:18:46
✔ Nitro built in 745 ms       
```

**Comment by danielroe** [comment]
Would you create an issue with a reproduction?

**Comment by m0ksem** [comment, marked as answer]
So I found that `nuxtOptions` in `getNitroContext` (packages/nitro/src/context.ts) have `dev` field set to true. Because of that, in `build` function (packages/nitro/src/build.ts) do not call `_build`, calling `_watch` instead.

So problem caused because `isDevelopment` from `std-env` package in `@nuxt/schema` is `true`. Likely, my NODE_ENV set globally to 'development'. 

The easiest way to fix that is to change build command to `"build": "NODE_ENV=production nuxi build"`.

Luckily, there are no issues with esbuild and other bundle stuff 😁.

I made a PR in https://github.com/nuxt/starter/pull/9, hope that can be helpful.
